lib/rules/no-invalid-double-slash-comments/index.cjs

```diff
@@ -37,9 +37,9 @@ const rule = (primary) => {
 		});
 
 		root.walkRules((ruleNode) => {
-			// ruleNode.selectors elements are trimmed
```

Comment:
I removed this comment because there is test coverage for this aspect and we already have guidance against using `node.selectors`.

Reply:
> we already have guidance against using `node.selectors`

I didn't know about that. If we already have a regression test coverage it can be removed.
i.e. the comment was there so that no one would refactor it back